src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -380,10 +381,18 @@ describe('<BudgetDetailPage />', () => {
         spentTransactions: { count: 0 },
       },
     });
+    useOfferRedemptions.mockReturnValue({
+      isLoading: false,
+      offerRedemptions: mockEmptyOfferRedemptions,
+      fetchOfferRedemptions: jest.fn(),
+    });
     renderWithRouter(<BudgetDetailPageWrapper />);
 
     // Catalog tab does NOT exist
     expect(screen.queryByText('Catalog')).toBeFalsy();
+
+    // Ensure no assignments-related empty states are rendered
+    expect(screen.queryByText('No budget activity yet? Assign a course!')).not.toBeInTheDocument();
```

Comment:
[context] Asserts that the overview empty state for the "Activity" tab no longer appears when feature flag is disabled.